massa-execution-worker/src/interface_impl.rs

```diff
@@ -47,6 +50,35 @@ impl InterfaceImpl {
     pub fn new(config: ExecutionConfig, context: Arc<Mutex<ExecutionContext>>) -> InterfaceImpl {
         InterfaceImpl { config, context }
     }
+
+    #[cfg(feature = "gas_calibration")]
+    /// Used to create an default interface to run SC in a test environment
+    pub fn new_default(
```

Comment:
Naming is a bit too generic? Maybe we can rename to new_for_gas_calibration or something explicit?

Reply:
Yes it's generic because the function is generic and could be used in different context. For the moment we only use for gas_calibration but could be useful in all cases where we want to have the execution module with default values.